tests/ui/feature-gates/feature-gate-object_safe_for_dispatch.stderr

```diff
@@ -62,7 +62,6 @@ LL | trait NonObjectSafe4 {
    |       -------------- this trait cannot be made into an object...
 LL |     fn foo(&self, s: &Self);
    |                      ^^^^^ ...because method `foo` references the `Self` type in this parameter
-   = help: consider moving `foo` to another trait
```

Comment:
I don't think this is a change we want. Note that this mentions a different `foo` than the previous messages.

Reply:
Ah okay, sorry about that, I pushed a different solution that doesn't change the other tests